### PR TITLE
WEBEU2 - User Interface - Nabeelah Yousuph 🇳🇬

### PR DESCRIPTION
**Pull request by NabeelahY**

### CODE REVIEW

File: great-idea-website/index.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
```
Comment (maximesalomon) [inline great-idea-website/index.html:7]: You might want to add `<meta name="viewport" content="width=device-width, initial-scale=1.0">` here

File: great-idea-website/index.html [hunk 2]
```diff
@@ -14,50 +15,89 @@
...
+      <a href='#'>Features</a>
+      <a href='#'>About</a>
+      <a href='#'>Contact</a>
+      <a href='#'><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+    </nav>
+
+  </header>
+
+  <section>
+    <h1>Innovation <br> On <br> Demand<div><button>Get Started</button></div>
```
Comment (maximesalomon) [inline great-idea-website/index.html:33]: Good use of semantic button tag

File: great-idea-website/index.html [hunk 3]
```diff
@@ -14,50 +15,89 @@
...
+      </section>
+
+      <section class="bottom-section">
+        <h4>Vision</h4>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse
+          eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+    </div>
+  </section>
```
Comment (maximesalomon) [inline great-idea-website/index.html:87]: you might want to use the `<main>` HTML5 tag to wrap the primary content :)

File: great-idea-website/css/index.css [hunk 4]
```diff
@@ -63,4 +63,73 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
```
Comment (maximesalomon) [inline great-idea-website/css/index.css:67]: You might want to add a container or stying to the body with some max-width to make sure your style looks good on bigger screen as well

File: great-idea-website/css/index.css [hunk 5]
```diff
@@ -63,4 +63,73 @@ h1, h2, h3, h4, h5 {
...
+	text-align: center;
+	font-size: 80px;
+	padding: 20px;
+	
+}
+button {
+	padding: 15px;
+	background-color: #fff;
+	width: 100px;
+}
```
Comment (maximesalomon) [inline great-idea-website/css/index.css:99]: Try to add some styling to match the design file

**Review by maximesalomon — APPROVED** [review]
Good job Nabeelah! Overall it's great, try to match the design file as precisely as possible. Almost pixel perfect is the goal. You git this :)

**Comment by NabeelahY** [comment]
Hello @maximesalomon. Thanks for the review. I've have added the corrections to the points you noted.

**Pull request merged**